FND/Relations/Relations.rdf

```diff
@@ -132,14 +130,13 @@ the FIBO FND RFC, was revised in advance of the September 2013 New Brunswick, NJ
 	<owl:ObjectProperty rdf:about="&fibo-fnd-rel-rel;defines">
 		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-rel-rel;represents"/>
 		<rdfs:label>defines</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-fnd-rel-rel;Reference"/>
 		<skos:definition>determines or identifies the essential qualities or meaning of, discovers and sets forth the meaning of, fixes or marks the limits of, demarcates</skos:definition>
 	</owl:ObjectProperty>
 	
 	<owl:ObjectProperty rdf:about="&fibo-fnd-rel-rel;designates">
 		<rdfs:label>designates</rdfs:label>
 		<rdfs:domain rdf:resource="&fibo-fnd-aap-agt;AutonomousAgent"/>
-		<skos:definition>to name something officially or appoint someone to a position officially</skos:definition>
+		<skos:definition>names something officially or appoints someone to a position officially</skos:definition>
```

Comment:
No no no.
These are entirely different semantics. They might use the same English word but must not use the same semantic property.

Reply:
Ok - I've limited this to naming something offucially, and appoints, as a subproperty makes more sense now

Reply:
No, how is "appoints" a subproperty of "names something officially"? An example of the latter might be to name the UK as "The United Kingdom of Grate Britain and Northern Ireland". 
Another reason it does not work for naming is that, if tracking the Authorizer (which this does as the domain of the property), and the object of the example triple is the country, then there seems no way to represent the official name that has been designated.

The other alternative "appoints someone" would make more sense for consistency, since the only uses of designates are as super property of a) appoints and b) delegatesControlTo.

My main concern with using "designates" in this way is that we use "designation" inconsistently to mean both "appointment" (consistent) and naming (inconsistent) - for example we have property preferredDesignation.
In summary I think we should stick to something like "appoints" here and remove the naming-related uses of "designation" as a separate issue.

Reply:
Ok - I will revise the definition of designates to be appoints someone, and then eliminate the subproperty 'appoints', since it is redundant given this change.